src/configs/index.js

```diff
@@ -8,3 +8,5 @@
 require('./mongodb')()
 
 require('./kafka')()
+
+require('@ankitpws/caching-library').RedisConfig()
```

Comment:
@ankitpws keep the name as node-cache or node-caching, it shouldn't include any user name.

Reply:
@rakeshSgr there are a lot of packages available on npm and this name is already taken so we can not name package as node-cache or node-caching instead we can make @sl/node-cache then we can publish package